CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Helm settings for external psql database were unused by backend (<https://github.com/openvinotoolkit/cvat/pull/3779>)
 - Updated WSL setup for development (<https://github.com/openvinotoolkit/cvat/pull/3828>)
 - Helm chart config (<https://github.com/openvinotoolkit/cvat/pull/3784>)
+- Fixed Interaction handler keyboard handlers (<https://github.com/openvinotoolkit/cvat/pull/3881>)
```

Comment:
1.7 has been just release, please update this file

Reply:
Done.